crates/ruff/src/importer.rs

```diff
@@ -18,10 +17,7 @@ pub struct Importer<'a> {
     python_ast: &'a Suite,
     locator: &'a Locator<'a>,
     stylist: &'a Stylist<'a>,
-    /// A map from module name to top-level `StmtKind::ImportFrom` statements.
-    import_from_map: FxHashMap<&'a str, &'a Stmt>,
```

Comment:
I opted to get rid of this and just do a full search. The number of imports is typically not huge, this operation is relatively rare, and we'd now have to track `FxHashMap<&'a str, Vec<&'a Stmt>>` anyway.